src/test/java/io/confluent/idesidecar/restapi/auth/CCloudOAuthContextTest.java

```diff
@@ -105,6 +106,82 @@ void createTokensFromAuthorizationCodeShouldExchangeControlAndDataPlaneTokens()
     }
   }
 
+  @Test
+  void exchangeControlPlaneTokenShouldFailIfCookieHeaderIsEmpty() throws Throwable {
+    var testContext = new VertxTestContext();
+    var authContext = new CCloudOAuthContext();
+
+    wireMock.register(
+        WireMock
+            .post("/api/sessions")
+            .willReturn(
+                WireMock
+                    .aResponse()
+                    .withHeader("Set-Cookie", "")
+                    .withStatus(201)
+                    .withBody("{}")
+            ));
+
+    var request = new ExchangeControlPlaneTokenRequest("valid_id_token", null);
+
+    authContext.exchangeControlPlaneToken(request)
+        .onComplete(
+            testContext.failing(failure ->
+                testContext.verify(() -> {
+                  assertEquals(
+                      "Empty cookie header string",
```

Comment:
the cookie parser doesn't identify anything wrong when the token field is an empty string, however, when the entire header is an empty string, we get a Java exception, so I have updated the test to reflect that